idaes/surrogate/pysmo/tests/test_sampling.py

```diff
@@ -189,6 +194,8 @@ def test_points_selection_01(self, array_type):
         input_array = array_type(self.test_data_3d)
         generated_sample_points = np.array([[-0.5, 10], [10, 100]])
         SamplingClass = SamplingMethods()
+        SamplingClass.data_headers = [i for i in range(0, generated_sample_points.shape[1] + 1)]
+        SamplingClass.x_data = np.zeros((2, input_array.shape[1] - 1))
```

Comment:
This piece of code seems to be repeated a few times. Would it make sense to refactor it to a common point? e.g.

```py
def _create_sampling(sample_points: np.ndarray, input_array: np.ndarray) -> SamplingMethods:
    sampling = SamplingMethods()
    sampling.data_headers = ...
    sampling.x_data = ...
    return sampling
```

Reply:
Done